[ao] making _is_activation_post_process private

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88404
* #88400
* #88399
* #88398
* #88397
* #88396
* #88395
* #88394
* #88393
* #88392
* #88391
* #87885
* #87883
* #87521
* __->__ #87520

Summary: same function in observer and quantize, consolidated to a
single function. Note the definitions were slightly different, I've
changed the definition to be maximally inclusive so that the name of the
function is more accurate

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

HDCharles committed:
activation_is_post_process
[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

cc jerryzh168 jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

cc jerryzh168 jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

cc jerryzh168 jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen

[ghstack-poisoned]

HDCharles committed:
Update on "activation_is_post_process"


[ao] making _is_activation_post_process private

Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

cc jerryzh168 jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

jcaip commented on torch/ao/quantization/quantize.py:
```diff
@@ -62,11 +62,6 @@ def get_default_custom_config_dict():
     """
     return _DEFAULT_CUSTOM_CONFIG_DICT
 
-def is_activation_post_process(module):
```
I saw that the observer [function](https://github.com/pytorch/pytorch/blob/133e61af7a8f4b098daf7d34f848e3c2a6cb4ae4/torch/ao/quantization/observer.py#L1437) and this different, just wanted to double check

HDCharles replied:
yeah, it added a few definitions that weren't captured so it will be more accurate unless people were using this as a negative test.

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709276).

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function. Note the definitions were slightly different, I've
changed the definition to be maximally inclusive so that the name of the
function is more accurate

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function. Note the definitions were slightly different, I've
changed the definition to be maximally inclusive so that the name of the
function is more accurate

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function. Note the definitions were slightly different, I've
changed the definition to be maximally inclusive so that the name of the
function is more accurate

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] making _is_activation_post_process private"


Summary: same function in observer and quantize, consolidated to a
single function. Note the definitions were slightly different, I've
changed the definition to be maximally inclusive so that the name of the
function is more accurate

Test Plan: python test/test_public_bindings.py
python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709276](https://our.internmc.facebook.com/intern/diff/D40709276)

[ghstack-poisoned]

bigfootjon commented:
@pytorchbot revert -m="Diff reverted internally" -c="ghfirst"

bigfootjon commented:
@pytorchbot revert -m="Diff reverted internally" -c="ghfirst"
